dev/bin/build.js

```diff
@@ -247,14 +266,15 @@ export async function main(argv) {
     try {
         await buildLibs();
         const variantNames = /** @type {string[]} */ ((
-            argv.length === 0 || args.get('all') ?
-            manifestUtil.getVariants().filter(({buildable}) => buildable !== false).map(({name}) => name) :
-            args.get(null)
+            // eslint-disable-next-line dot-notation
+            argv.length === 0 || args['all'] ?
+            manifestUtil.getVariants().filter(({buildable}) => buildable !== false).map(({name}) => name) : []
         ));
         await build(buildDir, extDir, manifestUtil, variantNames, manifestPath, dryRun, dryRunBuildZip, yomitanVersion);
     } finally {
         // Restore manifest
-        const manifestName = /** @type {?string} */ ((!args.get('default') && args.get('manifest') !== null) ? args.get('manifest') : null);
+        // eslint-disable-next-line dot-notation
+        const manifestName = /** @type {?string} */ ((!args['default'] && typeof args['manifest'] !== 'undefined') ? args['manifest'] : null);
```

Comment:
Suggestions:
- Just use dot notation
- Switch the arg descriptors from using `kebab-case` to `camelCase` for better idiomatic JS
- Can the type cast be changed to a declaration annotation, which should be safer?
  ```js
  /** @type {?string} */
  const manifestName = ...;
  ```

Reply:
Typescript is not too happy about the typecast
```
Type 'string | boolean | (string | boolean)[] | null' is not assignable to type 'string | null'.
  Type 'boolean' is not assignable to type 'string'.ts(2322)
```

Reply:
You can leave it as-is then, I wasn't sure how much typing util.parseArgs provided